modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -40,7 +40,7 @@ You can see the configured TLS security profile in the `APIServer` custom resour
 
 [NOTE]
 ====
-The control plane does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3`, it is not supported.
+The control plane does not support TLS `1.3` as the minimum TLS version and because the `Modern` profile requires TLS `1.3`, it is not supported.
```

Comment:
The phrase "it is not supported" can be confusing here, as it might mean the profile (likely) or TLS (unlikely). I know it's not part of the diff for this PR, but you might consider saying something like "..._the profile_ is not supported."

Reply:
Thanks for the thoughts - I thought this wasn't super clear either so I appreciate the kick to modify it!